i18n/en.json

```diff
@@ -0,0 +1,602 @@
+  "begin.ignored.title": "Begin Ignored",
+  "command": "command",
+  "cutoff.failure.invalidEndYear.description": "The end year must be after or equal to the start year.",
+  "cutoff.failure.invalidEndYear.title": "Invalid End Year",
+  "cutoff.help.description": "Set a cutoff year range for songs. If one value is specified, only songs AFTER that year will be played. If two values are specified, only songs BETWEEN those two years will be played.",
+  "cutoff.help.example.reset": "Reset to the default cutoff of {{{defaultBeginningSearchYear}}} to {{{defaultEndSearchYear}}}",
+  "cutoff.help.example.singleCutoff": "Play songs released after the year {{{year}}}",
+  "cutoff.help.example.twoCutoffs": "Play songs released between the years {{{beginningYear}}}-{{{endYear}}}",
+  "debug.description": "If you were asked by a bot developer to do this, give them this:\n{{{debugID}}}",
+  "debug.title": "Debug Details Sent!",
+  "duration.failure.removingDuration.notSet.description": "The duration is not currently set.",
+  "duration.failure.removingDuration.title": "Error Removing Duration",
+  "duration.failure.removingDuration.tooShort.description": "The duration cannot be less than 2 minutes.",
+  "duration.help.description": "Sets a maximum length for the KMQ game in minutes.",
+  "duration.help.example.decrement": "Remove {{{duration}}} minutes from the current game's duration",
+  "duration.help.example.increment": "Add {{{duration}}} minutes to the current game's duration",
+  "duration.help.example.reset": "Disables the duration",
+  "duration.help.example.set": "The game will automatically end after {{{duration}}} minutes",
+  "end.help.description": "Finishes the current game and decides on a winner.",
+  "exclude.failure.unrecognizedGroups.excluded": "excluded",
+  "exclude.help.description": "Select as many groups that you would like to ignore, separated by commas. A list of group names can be found [here]({{{groupList}}}).",
+  "exclude.help.example.multipleGroups": "Ignore songs from {{{groupOne}}}, {{{groupTwo}}}, and {{{groupThree}}}",
+  "exclude.help.example.reset": "Resets the exclude option",
+  "exclude.help.example.singleGroup": "Ignore songs from {{{group}}}",
+  "exp.activeModifiers": "🚀 Active Modifiers 🚀",
+  "exp.artistGroupGuessModePenalty": "Artist/Group Guess Mode Penalty",
+  "exp.bonusArtists": "Guessing songs by the daily bonus artists",
+  "exp.bonusArtistsTitle": "🎤 Current Bonus Artists 🎤",
+  "exp.bonusDescription": "You can get bonus EXP for the following",
+  "exp.bonusTitle": "Ways to get EXP Bonuses",
+  "exp.explanation.bonusArtistGuess": "Guessing correctly for a bonus artist",
+  "exp.explanation.firstGameOfDay": "First game of the day",
+  "exp.explanation.powerHour": "Playing during a KMQ Power Hour or Weekend",
+  "exp.explanation.quickGuess": "Guessing quickly",
+  "exp.explanation.rareGuess": "Rare correct guesses bonus",
+  "exp.explanation.rareGuess.upTo": "up to",
```

Comment:
Translations should be full strings, not partial strings. The grammar for these might be structured differently in different languages